service/src/main/resources/db/migration/V373__calendar_event_time_reservation.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE calendar_event_time
+(
+    id                uuid PRIMARY KEY                  DEFAULT ext.uuid_generate_v1mc(),
+    created           timestamp with time zone NOT NULL DEFAULT now(),
+    updated           timestamp with time zone NOT NULL DEFAULT now(),
+    modified_at       timestamp with time zone NOT NULL,
+    modified_by       uuid                     NOT NULL REFERENCES evaka_user (id),
+    calendar_event_id uuid                     NOT NULL REFERENCES calendar_event (id) ON DELETE CASCADE,
+    date              date                     NOT NULL,
+    start_time        time without time zone   NOT NULL,
+    end_time          time without time zone   NOT NULL,
+    child_id          uuid REFERENCES child (id)
+);
+
+CREATE TRIGGER set_timestamp
+    BEFORE UPDATE
+    ON calendar_event_time
+    FOR EACH ROW
+EXECUTE PROCEDURE trigger_refresh_updated();
```

Comment:
If you want to comply with the new naming pattern, I think you need to create a new procedure that updates `updated_at` column